docs/sources/style-guide/write-for-developers/index.md

```diff
@@ -0,0 +1,148 @@
+
+Additional tips:
+- Remember to write concisely. Don't say "This method does this..." or "This parameter describes that..."
+- When the names of code elements are singular, don't make them plural. Instead, add a plural noun to describe them. For example, don't change `MyEvent` to `MyEvents`; refer to the `MyEvent` objects. 
+- If the element does some sort of action, start the first sentence of the description with an action verb. 
+
+## Code examples
+
+It is common for readers of documentation to skim through it to find code samples they can copy and paste and run "as is". For this reason, it is worth taking the time to ensure that whenever possible the examples you provide are production-ready.
+
+Of course, this doesn't mean you should avoid writing code examples intended merely to illustrate a point. When your intent is to explain something, it isn't necessary for your code to be runnable so long as the developer can learn from it to do something similar on their own.
+
+When providing an example, give a written description. You can put it either in the body of the document or as explanatory comments within the example code.
+
+Remember the basic rule: your explanation of the code shouldn't describe _what_ it does, but _why_ it does what it does. (Refer to [_Docs for Developers_](https://docsfordevelopers.com/)).
+
+### Formatting code examples
+
+Here are guidelines to follow when formatting code examples.
+
+- Use spaces, not tabs. 
+- Follow Grafana's accepted [coding style guidelines](https://github.com/grafana/grafana/blob/main/contribute/style-guides/code-comments.md).)
+- Wrap lines at 80 characters.
+- When omitting code, use three dots (...). Don't use the ellipsis character (…).
+
+Introduce each code sample with a sentence or paragraph to establish its context. End the introduction with a colon if it immediately precedes the sample or a period if it doesn't. 
+
+### Paths, filenames and URLs
+
+There are many types of information that should be put in code font. Among these are filenames, paths, folders, and directories. However, don't format domain names or URLs as code if you intend the user to follow the link. 
+
+While there are times when you may want to make a URL clickable, it is better to follow our [guidelines for references]({{< relref "../../../writing-guide/references/" >}}).
+
+## Command lines
+
+Use the following conventions when you include commands on the command line in technical content.
+
+- Do not assume everyone is using Linux. Make sure instructions include enough information for Windows and Mac users to successfully complete procedures.
+
+- Do not add `$` before commands. Make it easy for users to copy and paste commands.
+
+  - **Right:** `sudo yum install grafana`
+  - **Wrong:** `$ sudo yum install grafana`
+
+- Include `sudo` before commands that require `sudo` to work.
+
+For terminal examples and Grafana configuration, use a `bash` code block:
```

Comment:
It would be helpful to include the syntax for the bash code block.  Remember, community members cannot go into the source code for this page to figure the syntax out because this is a private repository.

Reply:
The first place I would want to see that information is in the Markdown guide. Looking there, there's a section on code comments which gives an example in JavaScript. Correct me if I'm wrong, but I think the bash and console code blocks are formed in the same fashion. So, we could add to that section with more examples or put in an explanatory note.

That said, while I'm not opposed to adding that information into this section, I do think it may divert from the primary purpose of style to go into Markdown syntax.

Reply:
The markdown page works too.  I just like the ease of being able to copy/paste the syntax without having to go hunt around for it.  For reference, check out how I recently updated the Admonitions section.  
https://grafana.com/docs/writers-toolkit/style-guide/style-conventions/#admonitions